interfaces/apparmor/template_vars.go

```diff
@@ -28,11 +28,11 @@ import (
 
 // templateVariables returns text defining apparmor variables that can be used in the
 // apparmor template and by apparmor snippets.
-func templateVariables(appInfo *snap.AppInfo) []byte {
+func templateVariables(appName string, info *snap.Info) []byte {
 	var buf bytes.Buffer
-	fmt.Fprintf(&buf, "@{APP_NAME}=\"%s\"\n", appInfo.Name)
-	fmt.Fprintf(&buf, "@{SNAP_NAME}=\"%s\"\n", appInfo.Snap.Name())
-	fmt.Fprintf(&buf, "@{SNAP_REVISION}=\"%s\"\n", appInfo.Snap.Revision)
+	fmt.Fprintf(&buf, "@{APP_NAME}=\"%s\"\n", appName)
```

Comment:
A bit worried here as we're mixing app name and hook name on the same namespace with the same variable name. How is this variable being used today, and what is the potential for security issues?